api/python/setup.py

```diff
@@ -1,13 +1,13 @@
 import os
```

Comment:
I noticed in the contributing documentation that you had an invalid pip install command so I rearranged how the deps are setup and changed the docs to be more accurate. (At least from what I can tell)